src/gui/systray.h

```diff
@@ -94,7 +94,7 @@ class Systray
     void shutdown();
 
     void hideWindow();
-    void showWindow();
+    void showWindow(); // Listened to by the QML Window component
```

Comment:
this change seems unrelated

Reply:
I wracked my head for a while trying to figure out how this signal invoked the QML window until realising there was a connection in `Window.qml`. Can remove, but might prove useful for future contributors

Reply:
at least, makes it a separate commit
if we need to revert the fix of the tray icon that would also remove your comment
that is the main reason to not add unrelated changes
best would be a separate pr

Reply:
That is fair -- removed the comment